data_steward/tools/consolidated_reports/get_all_results_html.py

```diff
@@ -3,10 +3,14 @@
 import json
 import os
 
-import gcs_utils
+from gcloud.gcs import StorageClient
+import app_identity
 from tools.consolidated_reports import query_reports
 
-DRC_BUCKET_PATH = 'gs://%s/' % gcs_utils.get_drc_bucket()
+project_id = app_identity.get_application_id()
+storage_client = StorageClient(project_id)
+drc_bucket = storage_client.get_drc_bucket()
```

Comment:
@lledezma this would not work, as get_drc_bucket() is a local function in gcs_utils, you would have to re-implement get_drc_bucket() somewhere to get the bucket name from environment variables probably query_reports.py